test/fixtures.json

```diff
@@ -66,6 +66,13 @@
         "message": "dsa with 1024 keys and a password variant",
         "scheme": "DSA-SHA",
         "signature": "302c02147e0eb669879a758df5b3efae631fe51052852434021472e7d838db4356f9d5d305c78dbda65711259bfa"
+      },
+      {
+        "public": "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUhZd0VBWUhLb1pJemowQ0FRWUZLNEVFQUNJRFlnQUVXVUIzL0RwOVZSWGR5VjhZY3lqRVFVWGRkMEFzUHh3bwpQTVpBRzJQbG1SVlhuRmMyZWhuT0tOVDVxdklGeXJTb0JYS285ckNmZi9CN0kwR3hpNzBNOHVML0JRS2Z5MWpBCkRvSEp1WndWVXpMa0RkZU1lL2ZiK1dXNFF1YUFVbHBDCi0tLS0tRU5EIFBVQkxJQyBLRVktLS0tLQo=",
+        "private": "LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1JR2tBZ0VCQkRBME5VbVMzOHoxcU1QK2R6Sjk4Y3J4bFkzZXhzN0tOVVlqZ25zODZzZ0ZBTmhPd3YyT09uVVgKcTcvbHZZUW1QN1NnQndZRks0RUVBQ0toWkFOaUFBUlpRSGY4T24xVkZkM0pYeGh6S01SQlJkMTNRQ3cvSENnOAp4a0FiWStXWkZWZWNWelo2R2M0bzFQbXE4Z1hLdEtnRmNxajJzSjkvOEhzalFiR0x2UXp5NHY4RkFwL0xXTUFPCmdjbTVuQlZUTXVRTjE0eDc5OXY1WmJoQzVvQlNXa0k9Ci0tLS0tRU5EIEVDIFBSSVZBVEUgS0VZLS0tLS0K",
+        "message": "ECDSA with P-384",
+        "scheme": "ecdsa-with-SHA1",
+        "signature": "3066023100fb65dfe219093dbb7b468ccded39931d845ff73a2386b9fd7a6a9860c5cdc56f120370dfa6b0441e14d30e5bfca371740231008f603c40136c059eb8f2de65fb9e773a79ed59102dcc1bfcc863c254f93a8139d579f952fa9e9512ba4ef75660b7565a"
```

Comment:
Can we add descriptions to these fixtures?
Hard to tell why they are important.